src/Views/OperatingSystemView.vala

```diff
@@ -567,6 +593,18 @@ public class About.OperatingSystemView : Gtk.Box {
         }
     }
 
+    public string get_progress_text () {
+        if (download_size_max == 0) {
+            return "";
+        }
+
+        uint64 downloaded_size = download_size_max - download_size_remaining;
+        double downloaded_mb = (double) downloaded_size / (1024 * 1024);
+        double total_mb = (double) download_size_max / (1024 * 1024);
+
+        return "%.2f MB / %.2f MB".printf (downloaded_mb, total_mb);
```

Comment:
Should use https://valadoc.org/glib-2.0/GLib.format_size.html probably instead of assuming the file size will be in megabytes